docs/guide/guide_to_ez_platform.md

```diff
@@ -1,11 +0,0 @@
-# Guide to eZ Platform
-
-## A comprehensive, topical reference manual from A to Z
-
-Whether you're just starting out or already a veteran, the **Guide to eZ Platform** has something for you.
-
-It contains a thorough description of *everything* about eZ Platform that you need to know as a developer, beginning with an explanation of underlying concepts and philosophy, to a presentation of all important features of the system.
```

Comment:
Because it doesn't bring any value: it's textual padding.
Plus, I'm not sure if claiming it's a description of "Everything in eZ Platform" isn't a tiny bit of an exagerration